### PR TITLE
Niranjan

### PR DESCRIPTION
**Pull request by niranjanhegde144**
Added Contributors page. Might need smaller modifications.

### CODE REVIEW

File: src/components/contributors.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
+import React from "react";
 
 let mapDetail = (detail) => {
   console.log(detail);
```
Comment (akramkazmi71) [inline src/components/contributors.js:4]: If testing is done, remove this statement.

File: src/App.js [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+import "./App.css";
+import Home from "./components/home.js";
+import Events from "./components/event.js";
+import Programming from "./components/programming.js";
+import Result from "./components/result.js";
+import Contributors from "./components/contributors.js";
+import details from "./contributors_details.json";
 
 function App() {
+  console.log(details);
```
Comment (akramkazmi71) [inline src/App.js:12]: If testing is done, remove this statement.

File: src/components/Competetive.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+export default function Competetive() {
+  return (
+    <div>
+      <form action="#" method="POST">
+        <input type="file" name="solution" id="solution" />
+      </form>
+    </div>
+  );
+}
```
Comment (akramkazmi71) [inline src/components/Competetive.js:11]: I have created the file programming for this. Are we going forward with this file then?
If yes, then delete the 'programming.js' file from components.

File: src/components/Contributors.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+let mapDetail = (detail) => {
+  console.log(detail);
```
Comment (akramkazmi71) [inline src/components/Contributors.js:4]: Remove if not necessary.

File: src/index.js [hunk 5]
```diff
@@ -1,16 +1,10 @@
...
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
 
-ReactDOM.render(
-  <BrowserRouter>
-    <App />
-  </BrowserRouter>,
-  document.getElementById('root')
-);
+ReactDOM.render(<App />, document.getElementById("root"));
```
Comment (akramkazmi71) [inline src/index.js:7]: Is router working after removing BrowserRouter?

Comment (niranjanhegde144) [reply src/index.js:7]: I haven't checked it. I'll change it as before.

**Pull request merged**